packages/sockethub-schemas/src/object-types.js

```diff
@@ -54,6 +54,9 @@ const objectTypes = {
       },
       "content": {
         "type": "string"
+      },
+      "published": {
+        "enum": ["this is not a timestamp"]
```

Comment:
I'm a little confused as to why this is an enum of `this is not a timestamp` ?

Reply:
https://github.com/sockethub/sockethub/pull/691#issuecomment-1051270745